### PR TITLE
[egeria-connector-jdbc] initial commit

### PR DESCRIPTION
**Pull request by lcpopa**
Signed-off-by: Liviu Popa <liviu-constantin.popa@ing.com>

### CODE REVIEW

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            "Reading omas information",
+            "Consult logs for further details"),
+    UNKNOWN_ERROR_WHILE_METADATA_TRANSFER("JDBC-CONNECTOR-0009",
+            OMRSAuditLogRecordSeverity.EXCEPTION,
+            "Unknown error transferring metadata in method {0}.",
+            "Reading metadata information",
+            "Consult logs for further details");
+
+
+    private final AuditLogMessageDefinition messageDefinition;
```
[the remark this answers is not in the document]
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java:70]: Not relevant

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+                    ERROR_READING_JDBC.getMessageDefinition(methodName, sqlException.getMessage()), sqlException);
+        }
+        return new ArrayList<>();
+    }
+
+    private List<DatabaseTableElement> getOmasTables(String schemaGuid){
+        String methodName = "getOmasTables";
+        try{
+            return Optional.ofNullable(databaseIntegratorContext
+                    .getTablesForDatabaseSchema(schemaGuid, 0, 0)).orElseGet(ArrayList::new);
```
[the remark this answers is not in the document]
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java]: Outdated

[the remark this answers is not in the document]
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java]: Outdated

File: egeria-connector-jdbc/build.gradle [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+// Artifact names are taken from the directory by default, set in settings.gradle to override
+// The 'name' for the maven artifact, and description are set here
+ext.name = 'JDBC Connector'
+description = 'JDBC Connector for Egeria'
+
+dependencies {
```
Comment (planetf1) [inline egeria-connector-jdbc/build.gradle:11]: I'm thinking we need some refactoring to import the main Egeria pom (to get consistent versions) - but I can help with that in a later issue/pr :-)

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+ * <ul>
+ *     <li>Log Message Id - to uniquely identify the message</li>
+ *     <li>Severity - is this an event, decision, action, error or exception</li>
+ *     <li>Log Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the situation</li>
+ *     <li>UserAction - describes how a user should correct the situation</li>
+ * </ul>
+ */
+public enum JdbcConnectorAuditCode implements AuditLogMessageSet {
+
```
Comment (planetf1) [inline egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java:22]: I think a lot of these messages are very code-centric. I think as we develop & get more experience with the connector we can later improve/refactor to have better end-user recommendations & clarity of overall impact. However as this is a first step I think it's fine to get started...

**Review by planetf1 — APPROVED** [review]

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java (hunk 4 above)
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java:22]: #180

**Review by lpalashevski — APPROVED** [review]
Approved with remark that this is good initial implementation, we will probably need to further discuss some aspects like qualified name patterns for the db and related elements.

**Pull request merged**